cnf-tests/entrypoint/test-run.sh

```diff
@@ -2,20 +2,6 @@
 set -e
 # Setting -e is fine as we want both config and validiation to succeed
 # before running the "real" tests.
-
-TEST_SUITES=${TEST_SUITES:-"validationsuite configsuite cnftests"}
-SUITES_PATH="${SUITES_PATH:-~/usr/bin}"
-
-suites=( $TEST_SUITES )
-if [ "$IMAGE_REGISTRY" != "" ] && [[ "$IMAGE_REGISTRY" != */ ]]; then
-    export IMAGE_REGISTRY="$IMAGE_REGISTRY/"
-fi
-
-for suite in "${suites[@]}"; do
-    if [ "$DISCOVERY_MODE" == "true" ] &&  [ "$suite" == "configsuite" ]; then
-        echo "Discovery mode enabled, skipping setup"
-        continue
-    fi
-    echo running "$SUITES_PATH/$suite" "$@"
-    "$SUITES_PATH/$suite" "$@"
-done
+pushd /usr/bin/latency_testing/one/two/three/four/five/
+LATENCY_TEST_RUN=true DISCOVERY_MODE=true FEATURES=performance ./5_latency_testing.test --ginkgo.no-color --ginkgo.vv --ginkgo.flake-attempts 2 -test.failfast --ginkgo.focus "\[performance\]\ Latency\ Test" --ginkgo.junit-report ~/tmp/artifacts
```

Comment:
Don't we want to invoke `latency-e2e.test` as well?

Reply:
In the origin `test-run.sh` we had: `"$SUITES_PATH/$suite" "$@"` which passed to the test binary
all the parameters we invoked the `test-run.sh` with.
This allowed us to pass the --junit and --report flags and chose where to output the reports.
Here you hardwire only the default `ginkgo.junit-report` to ~/tmp/artifacts. 
Should we keep the `$@` so that the user could specify values for the junit and reporter flags?

Reply:
Not only that but after this change, clients should be able to do the same as before, by the [documentation](https://docs.openshift.com/container-platform/4.13/scalability_and_performance/cnf-performing-platform-verification-latency-tests.html):
1. Running hwlatdetect
2. Running cyclictest
3. Running oslat
4. Generating a latency test failure report
5. Generating a JUnit latency test report
6. Running latency tests on an SNO
7. Running latency tests in a disconnected cluster
 
 So I think the `$@` is necessary for more than just the reports.
 And regarding all the requirements (1-7) we should remember to test
 if each one is working as usual after this change.

Reply:
After a talk with @shajmakh I have found I was approaching this the wrong way, so forget about my answer.
I need to rework the PR

Reply:
ok, sounds good! thanks for the update.

Reply:
> 

yeah that would not be a problem